### PR TITLE
responding to my own comments on jp's branch

### PR DESCRIPTION
**Pull request by lbergelson**
since @jean-philippe-martin is away I'm responding to my own comments on https://github.com/broadinstitute/gatk/pull/3159

### CODE REVIEW

**Comment by lbergelson** [comment]
@droazen Want to take a look at my commit in this?

File: build.gradle [hunk 1]
```diff
@@ -126,6 +126,10 @@ dependencies {
     compile 'com.google.cloud:google-cloud-nio:0.20.4-alpha-20170727.190814-1:shaded'
     compile 'com.google.cloud.genomics:google-genomics-dataflow:v1beta2-0.15'
     compile 'com.google.cloud.genomics:gatk-tools-java:1.1'
+    // this comes built-in when running on Google Dataproc, but the library
+    // allows us to reads from GCS also when testing locally (or on non-Dataproc clusters,
```
Comment (droazen) [inline build.gradle]: reads -> read

File: src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/PrintReadsSparkIntegrationTest.java [hunk 2]
```diff
@@ -47,6 +47,36 @@ public String getTestedClassName() {
...
     }
 
+    @DataProvider
+    public Object[][] gcsTestingData() {
+        return new Object[][]{
+            {"org/broadinstitute/hellbender/engine/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.10000000-10000020.with.unmapped.bam",
+                ".bam", false, null},
+// TODO: uncomment once we upgrade to the version of gcloud that includes JP's fix (#2124)
+//            {"org/broadinstitute/hellbender/engine/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.10000000-10000020.with.unmapped.bam",
+//                ".bam", true, null},
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/PrintReadsSparkIntegrationTest.java]: Can we uncomment this yet @lbergelson ?

**Comment by lbergelson** [comment]
@droazen Test enabled.

**Pull request merged**